### DISCUSSION TITLE
Has anyone had any luck with any bitmap font format?

### DISCUSSION THREAD

**Discussion by JackNWhite** [zed-industries/zed#13649] [Feature Requests]
It's all in the question. I have both the PCF and OTB formats of Terminus installed, but neither works in Zed. Is there another, working, format that can be used, or does it just not work?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
